### ISSUE TITLE
Question about Classes Index

### ISSUE THREAD

**Issue by lxa9867** [Epiphqny/VisTR#11]
Thank you for sharing this great work! I have a question about your classes index. YoutubeVIS has 40 classes and a empty class is involved in VisTR, thus 41 classes in total, right? Then I noticed that in your implementation, the loss_labels() is computed by 
``` 
target_classes = torch.full(src_logits.shape[:2], self.num_classes, dtype=torch.int64, device=src_logits.device)
target_classes[idx] = target_classes_o
loss_ce = F.cross_entropy(src_logits.transpose(1, 2), target_classes, self.empty_weight)
``` 
Since ytvis's class begins from 1 to 40 and I didn't find any re-index operation in your code, I am confused by the empty class here. Do you use 40 represent empty class (you set self.num_classes=40)? Thank you

**Comment by Epiphqny** [comment]
Hi @LiXiAng9867, the empty class is 0.

**Comment by LiXiAng9867** [comment]
Thank you for your reply. I am still confused... If empty class is 0, why you fill 40 to empty objects when compute ce loss? 
```
target_classes = torch.full(src_logits.shape[:2], self.num_classes, dtype=torch.int64, device=src_logits.device)
```

**Comment by Epiphqny** [comment]
@LiXiAng9867 Thanks for pointing out this issue, I will check this problem and reply later.

**Comment by Epiphqny** [comment]
Hi @lxa9867, thanks for your question, I fixed the bug and change the class number to 41, the result improves from 0.38 to 0.40, I will update the code and result later.

**Comment by lxa9867** [comment]
That's great. Hope for your new results!

**Comment by Epiphqny** [comment]
hi @lxa9867, I have fixed the bug and updated the code and result.
